pkgs/games/nxengine-evo/default.nix

```diff
@@ -10,9 +11,11 @@
 stdenv.mkDerivation rec {
   pname = "nxengine-evo";
   version = "2.6.4";
-  src = fetchurl {
-    url = "https://github.com/nxengine/nxengine-evo/archive/v${version}.tar.gz";
-    sha256 = "1xir74l0vrzrpyl7sfqfxp083dakbix26hd0arwf2y57w1mbjas8";
+  src = fetchFromGitHub {
+    owner = "nxengine";
+    repo = "nxengine-evo";
+    rev = "v${version}";
+    sha256 = "sha256-krK2b1E5JUMxRoEWmb3HZMNSIHfUUGXSpyb4/Zdp+5A=";
```

Comment:
Why does the hash change?

```
nix-hash --type sha256 --to-base16 1xir74l0vrzrpyl7sfqfxp083dakbix26hd0arwf2y57w1mbjas8
482bb96ae0a778e17856a041237a5c53b581c0ed0e3b7da8bff9e70d283939f6

nix-hash --type sha256 --to-base16 sha256-krK2b1E5JUMxRoEWmb3HZMNSIHfUUGXSpyb4/Zdp+5A=
92b2b66f513925433146811699bdc764c3522077d45065d2a726f8fd9769fb90
```

Reply:
I think the reason for this is `fetchurl` stores a file exactly, `fetchFromGitHub` decompresses it.

Reply:
Ah, that makes sense. This looks good to me then.